coalib/parsing/DefaultArgParser.py

```diff
@@ -187,6 +187,10 @@ def default_arg_parser(formatter_class=None):
         help='filters `--show-bears` by the given languages')
 
     outputs_group.add_argument(
+        '--filter-by', nargs='+',
```

Comment:
normally I would say that this would be good and intuitive, but I'm not sure how it plays well with other arguments. coala takes quite many currently, and I have the feeling that this doesn't work any more:
```
# The args here don't make sense together, but just for what I mean:
coala --filter-by filter1 filter2 --non-interactive --save
```
But I recall that argparse might still handle that properly... could you test that? :)